### ISSUE TITLE
new subfolder shows no content starting with Hogfather

### ISSUE THREAD

**Issue by tracing-home** [splitbrain/dokuwiki#3152]
In an existing structure with subfolders, I created a new one in parallel to others that work fine. They existed before the Hogfather upgrade. 

The first time entering the folder the content shows (it was copied from elsewhere via terminal), after the page reloads, no pages/entries (with acmenu) will show. 

I tried creation of the folder via interface as well as terminal. Permissions and ownership are identical on all folder/files

Using Release rc2-2020-06-01 "Hogfather" RC2, all plugins are updated, searchindex manager ran delete and rebuild. Running on Linux using a farm structure

When entering the folder `/brot-rezepte/banh-mi/start`, it has the problem, apache error.log gets this error. Other folders preexisting to upgrade cause no error:

`
[Sat Jun 06 21:59:18.318683 2020] [php7:error] [pid 15524] [client ::1:32774] PHP Fatal error:  Uncaught Error: Cannot access protected property Doku_Handler::$status in /var/www/dokuwiki/lib/plugins/numberedheadings/syntax.php:133\nStack trace:\n#0 /var/www/dokuwiki/inc/parser/handler.php(240): syntax_plugin_numberedheadings->handle('=== - Mixing ==...', 5, 3202, Object(Doku_Handler))\n#1 /var/www/dokuwiki/inc/Parsing/Lexer/Lexer.php(278): Doku_Handler->plugin('=== - Mixing ==...', 5, 3202, 'numberedheading...')\n#2 /var/www/dokuwiki/inc/Parsing/Lexer/Lexer.php(205): dokuwiki\\Parsing\\Lexer\\Lexer->invokeHandler('=== - Mixing ==...', 5, 3202)\n#3 /var/www/dokuwiki/inc/Parsing/Lexer/Lexer.php(155): dokuwiki\\Parsing\\Lexer\\Lexer->dispatchTokens('', '=== - Mixing ==...', '_plugin_numbere...', 3202, 3202)\n#4 /var/www/dokuwiki/inc/Parsing/Parser.php(109): dokuwiki\\Parsing\\Lexer\\Lexer->parse('\\n\\nMix bread flo...')\n#5 /var/www/dokuwiki/inc/parserutils.php(216): dokuwiki\\Parsing\\Parser->parse('\\n====== Banh mi...')\n#6 /var/www/dokuwiki/inc/parserutils.php(181): p_get_instructions('====== Banh mi ...')\n#7 /var/www/dok in /var/www/dokuwiki/lib/plugins/numberedheadings/syntax.php on line 133, referer: http://kochen.localhost/brot-rezepte/start
`

**Comment by phy25** [comment]
Could you disable `numberedheadings` and see if it works? If yes, it's a duplicate of https://github.com/ssahara/dw-plugin-NumberedHeadings/issues/13. Please add more information there.

**Comment by tracing-home** [comment]
I tried that, but to no effect. I would think that my error occurs 
before the pages are read, because the sidebar is not populated 
(acmenu). Those pages concerned here do not use numbered headings.

While trying to find an entry that used numbered headings I now noticed 
another folder not displaying. Furthermore, running SearchIndex Manager 
again, it stalls on one of the pages. At that moment this shows in 
apache error.log, it is the same folder I could not read before:

`[Sun Jun 07 06:29:15.143067 2020] [php7:error] [pid 6013] [client 
::1:36684] PHP Fatal error:  Uncaught Error: Cannot access protected 
property Doku_Handler::$status in 
/var/www/dokuwiki/lib/plugins/numberedheadings/syntax.php:133\nStack 
trace:\n#0 /var/www/dokuwiki/inc/parser/handler.php(240): 
syntax_plugin_numberedheadings->handle('=== - Mixing ==...', 5, 3202, 
Object(Doku_Handler))\n#1 
/var/www/dokuwiki/inc/Parsing/Lexer/Lexer.php(278): 
Doku_Handler->plugin('=== - Mixing ==...', 5, 3202, 
'numberedheading...')\n#2 
/var/www/dokuwiki/inc/Parsing/Lexer/Lexer.php(205): 
dokuwiki\\Parsing\\Lexer\\Lexer->invokeHandler('=== - Mixing ==...', 5, 
3202)\n#3 /var/www/dokuwiki/inc/Parsing/Lexer/Lexer.php(155): 
dokuwiki\\Parsing\\Lexer\\Lexer->dispatchTokens('', '=== - Mixing 
==...', '_plugin_numbere...', 3202, 3202)\n#4 
/var/www/dokuwiki/inc/Parsing/Parser.php(109): 
dokuwiki\\Parsing\\Lexer\\Lexer->parse('\\n\\nMix bread flo...')\n#5 
/var/www/dokuwiki/inc/parserutils.php(216): 
dokuwiki\\Parsing\\Parser->parse('\\n====== Banh mi...')\n#6 
/var/www/dokuwiki/inc/parserutils.php(181): p_get_instructions('====== 
Banh mi ...')\n#7 /var/www/dok in 
/var/www/dokuwiki/lib/plugins/numberedheadings/syntax.php on line 133, 
referer: http://kochen.localhost/start?do=admin&page=searchindex
`

**Comment by phy25** [comment]
Sorry, I should have mentioned that you need to move `numberedheadings` out of plugin folder. Not sure how disabling a plugin work but looks like the second error log you paste still loads `numberedheadings`.

**Comment by tracing-home** [comment]
Yep, that did the trick now. Disabling didn't work, moving it out works. 
SearchIndex Manager now works again too. Interestingly enough now there 
are only 82 pages to index. Before (with |numberedheadings) a bit over a 
100.
|
